### PR TITLE
Feat auth

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Added Authentication feature to user login and logout to the video library

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,7 +26,9 @@ ReactDOM.render(
             <WatchLaterContextPro>
               <LikeVideoContext>
                 <HistoryContextProvider>
+                  <AuthContextProvider>
                   <App />
+                  </AuthContextProvider>
```
Comment (rkapoor10) [inline src/index.js]: maybe you should consider placing auth provider at the top...below the browser router

**Review by rkapoor10 — APPROVED** [review]
Everything looks fine. The code is clean. You can merge it.

File: src/assests/styles/auth.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+     align-items: center;
+}
+
+.input-block-name{
+     display: flex;
+}
+
+.input-block-check{
+     font-size: 1.2rem;
+     display: flex;
```
Comment (paarthbajaj) [inline src/assests/styles/auth.css:57]: Could make a utility class for repetitive CSS properties like flex.

**Review by paarthbajaj — APPROVED** [review]
Code looks good and structured. You can merge.

File: src/assests/styles/auth.css (hunk 2 above)
Comment (Kothalkarpriya) [reply src/assests/styles/auth.css:57]: ok

File: src/index.js (hunk 1 above)
Comment (Kothalkarpriya) [reply src/index.js]: ok

**Pull request merged**